bin/dcsa.ts

```diff
@@ -24,4 +25,7 @@ new DBStack(app, 'db', {
     dbSnapshotID:  process.env.DBSNAPSHOTID ?? "",
 })
 
+new DCSAProviderCtkStack(app, 'st', { "hostedZoneId": process.env.HOSTEDZONEID ?? "", "baseUrl": process.env.BASEURL ?? "localhost")
```

Comment:
The stack name/id 'st' has to be different than the DCSAStack, you could call it 'ctkst'. And ind the release_provider file you have to refer to 'ctkst', check comment there